### PR TITLE
Add a module that reports the current selinux state

### PR DESCRIPTION
**Pull request by bstinsonmhk**
I've tested this on my CentOS 7 workstation and on a Fedora workstation. Happy
to take any comments!

### CODE REVIEW

File: py3status/modules/selinux_state.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            else:
+                selinuxstring = 'permissive'
+                color = 'color_bad'
+        except OSError:
+            selinuxstring = 'disabled'
+            color = 'color_bad'
+
+        response = {
+            'cached_until': time() + self.cache_timeout,
+            'full_text': 'selinux: {0}'.format(selinuxstring),
```
Comment (ultrabug) [inline py3status/modules/selinux_state.py]: can you make this configurable via a `format` parameter plz ? see issue #98 

you can see it done on various other modules mate

File: py3status/modules/selinux_state.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Configuration parameters:
+    - cache_timeout : how often we refresh this module in seconds (10s default)
+
+Requires:
+    - libselinux-python
+    - libselinux-python3 (optional for python3 support)
+
+@author bstinsonmhk
+@license MIT
```
Comment (ultrabug) [inline py3status/modules/selinux_state.py]: would you please be okay for BSD like all other modules ?

**Comment by ultrabug** [comment]
Thanks a lot @bstinsonmhk this is a nice module to have ! Please see my comments on code. Cheers

**Comment by ultrabug** [comment]
ping @bstinsonmhk, please just at least tell me if you're okay with BSD license as I need your consent, I can do the rest after merging np :)

**Comment by bstinsonmhk** [comment]
@ultrabug: I think I have everything wrapped up, let me know if there's anything else I should wrap in.

**Comment by ultrabug** [comment]
Thanks a lot @bstinsonmhk ! FYI I'll do some QA commit and rename the module **selinux** so that it can be extended if needed later :)

**Comment by ultrabug** [comment]
All done, cheers :)

**Pull request merged**